model/utils.py

```diff
@@ -173,8 +184,9 @@ def train_one_epoch(train_loader, model, device, optimizer, criterion, perceptua
             else:
                 pass
             log_figure(step, output, spec, att, length, log_save_dir, args)
-            out_log = "step {}: train_loss {}; spec_loss {}; ".format(step,
-                                                                      losses.avg, spec_losses.avg)
+            mcd = get_MCD(spec,output,args.sampling_rate)
```

Comment:
it is in the "if step % args.train_step_log == 0:", which cannot monitor all the training sample

Reply:
BTW, put the mcd in training is OK, but I would prefer it in evaluation and test stage.